src/themes/money/theme.json

```diff
@@ -1304,6 +1304,18 @@
           },
           "boxShadow": "0px 2px 3px rgba(24, 24, 61, 0.06), 0px 5px 5px rgba(24, 24, 61, 0.06), 0px 8px 7px rgba(24, 24, 61, 0.06)",
           "borderRadius": "md"
+        },
+        "panel-primary-text-align-center": {
+          "variant": "elements.rich-text-block.variants.panel-primary",
+          "textAlign": ["center", "center"],
```

Comment:
does it need to be split between mobile/desktop if both are center?

Reply:
ive already updated this, it was copied from Uswitch theme.